entity-framework/ef6/index.md

```diff
@@ -25,7 +25,7 @@ EF6 implements many popular O/RM features:
 - A "Code First" experience to create entity models by writing code.
 - Models can either be generated from existing databases and then hand-edited, or they can be created from scratch and then used to generate new databases.
 - Integration with .NET Framework application models, including ASP.NET, and through databinding, with WPF and WinForms.
-- Database connectivity based on ADO.NET and numerous providers available to connect to SQL Server, Oracle, MySQL, SQLite, PostgreSQL, DB2, etc.
+- Database connectivity based on ADO.NET and numerous [providers](~/ef6/fundamentals/providers/index.md) available to connect to SQL Server, Oracle, MySQL, SQLite, PostgreSQL, DB2, etc.
```

Comment:
Should use xref with uid to refer to pages in the same repo.

Reply:
I'll fix the ones here, but ef6 docs are full of these

Reply:
No need to update all. Just new ones we should make sure are locale/redirect compatible.